pkgs/by-name/wi/wireguard-go/package.nix

```diff
@@ -54,11 +50,6 @@ buildGoModule (
       mv $out/bin/wireguard $out/bin/wireguard-go
     '';
 
-    passthru.tests.version = testers.testVersion {
```

Comment:
Why? It’s useful to determine whether the binary actually runs.

Reply:
Oh, okay, so I'll leave that in.